scalatest/src/main/scala/org/scalatest/tools/ScalaTestAntTask.scala

```diff
@@ -774,7 +774,7 @@ class ScalaTestAntTask extends Task {
    * Sets value of the <code>testngsuites</code> attribute.
    */
   def setTestNGSuites(testNGSuitePath: Path): Unit = {
-    for (element <- testNGSuitePath.list)
+    for (element <- testNGSuitePath.list.map(_.nn))
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.